ProcessMaker/processmaker
# FOUR-5155: Add to the "garbage collector" a way to handle duplicated scheduled tasks

## Issue & Reproduction Steps
The issue is hard to reproduce. To simulate the problem it is necessary to create a process  with a timer event, run a new request until it arrives to the timer event and manually run queries to duplicate rows in the scheduled-tasks table. To facilitate this, a process and the content of example data for the tables process_request, process_request_tokens and scheduled tasks are attached:
[to_reproduce.zip](https://github.com/ProcessMaker/processmaker/files/7907017/to_reproduce.zip)

_For the database run mysql -u dbuser -p databaseName < example_data.sql_

## Solution
- A new process that handles the condition and clean the duplicated rows has been added to the command processmaker: garbage-collect

## How to Test
- Replicate the issue as described in the initial section of this PR
- From the command line run:  php artisan processmaker:garbage-collect
- Verify that the duplicated rows are removed from scheduled_tasks and process_request_tokens tables.

## Related Tickets & Packages
[https://processmaker.atlassian.net/browse/FOUR-5155](https://processmaker.atlassian.net/browse/FOUR-5155)

## Code Review Checklist
